.gitignore

```diff
@@ -1,2 +1,4 @@
 .idea/
-playbooks/chacal/
\ No newline at end of file
+.gitignore
+playbooks/chacal/
+playbooks/tjielp/
```

Comment:
I don't think it is sustainable to keep adding personal playbooks here. Ignore the whole playbooks directory? I should move my own playbook out of this repo also.

Reply:
Agree. I guess we could remove the whole playbooks directory and create an instruction in README to copy example-boat.yml to the local computer. Provide instructions on how to tailer that file and use it for provisioning. Having this file locally would also remove the need to use mechanisms like ansible vault as there is no risk of sharing secrets through git.
If this is ok i will see if i can make these changes (in a separate PR, quite new to this, i was already surprised that each commit was added to that single PR, will have to check how i create separate PRs). 

Additionally, i think it would be useful to have a backup script at the end of the installation instructions. After the user has configured signalk (plugins, settings, etc.). This could copy the relevant files (like default/settings/security.json) back to the local directory where the playbook is stored, to ease the next deployment.

Reply:
FWIW, I removed my (outdated) playbooks.

Reply:
When i place my playbook outside the playbooks folder the provisioning will fail because it uses relative paths for the handlers (like ../../main.yml). I guess using absolute paths will also not work. How did you use the playbook?

Reply:
Just keep the playbooks directory. Git does not version directories, so there needs to be at least one file there, so we can leave an example there.

Reply:
Moved example-boat.yml to /playbooks. Tested it and updated the README.
All commits in this PR should now be ok to be merged i think.